test/integration/custom-routes/test/index.test.js

```diff
@@ -320,6 +320,13 @@ const runTests = (isDev = false) => {
     )
   })
 
+  it('should handle multiple Set-Cookie', async () => {
+    const res = await fetchViaHTTP(appPort, '/multi-set-cookie')
+    expect(res.headers.get('Set-Cookie')).toBe(
+      'firstCookie=cookie1; Path=/, secondCookie=cookie2; Path=/'
```

Comment:
Note that this is not explicitly checking that two distinct `Set-Cookie` headers are being sent. It looks like this `fetchViaHTTP()` function is using `node-fetch` under the hood, so you could use `res.headers.raw()` to get the full array and validate against that instead.